static/sass/_pattern_navigation.scss

```diff
@@ -0,0 +1,18 @@
+@mixin canonical-p-navigation {
+  .p-navigation {
+    background-color: #2c001e;
+
+    .p-navigation__links > .p-navigation__link > a {
+      color: $color-x-light;
+
+      &:hover {
+        color: $color-mid-dark;
```

Comment:
Against the aubergine background, this hover state color fails the color contrast ratio accessibility test: https://webaim.org/resources/contrastchecker/?fcolor=666666&bcolor=2C001E

It's a hover state, so there's an argument to be made that any user with accessibility needs has likely hovered over the element because they intend to click on it, but there's a chance that isn't the case and [the guidance](https://webaim.org/articles/contrast/#changes) recommends maintaining an accessible color contrast in all states.